app/assets/javascripts/app/views.js

```diff
@@ -103,28 +103,27 @@ app.views.Base = Backbone.View.extend({
   },
 
   report: function(evt) {
-    if(evt) { evt.preventDefault(); }
-    var msg = prompt(Diaspora.I18n.t('report.prompt'), Diaspora.I18n.t('report.prompt_default'));
-    if (msg == null) {
-      return;
-    }
-    var data = {
-      report: {
-        item_id: this.model.id,
-        item_type: $(evt.currentTarget).data("type"),
-        text: msg
-      }
+    if (evt) { evt.preventDefault(); }
+    var report = {
+      item_id: this.model.id,
+      item_type: $(evt.currentTarget).data("type"),
+      text: "Not filled"
     };
+    document.getElementById("report-content-form").addEventListener("submit", function(event) {
```

Comment:
This is "old school javascript", I guess that function should be declared outside of the scope of this one, But I'm still not comfortable with backbone. Suggestions welcomed.